### PR TITLE
Kill all the integration test jobs in parallel

### PR DESCRIPTION
**Pull request by AlCutter**
Speeds up the integration test a bit by killing the jobs in parallel.

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: integration/functions.sh [hunk 1]
```diff
@@ -66,23 +66,32 @@ pick_unused_port() {
...
-    if ! ps -p ${pid} > /dev/null ; then
+    local im_still_alive=""
+    for pid in ${pids}; do
+      if ps -p ${pid} > /dev/null ; then
+        # https://www.youtube.com/watch?time_continue=1&v=VuLktUzq23c
+        im_still_alive+=" ${pid}"
+      fi
+    done
+    pids="${im_still_alive}"
+    if [ "${pids}x" == "x" ]; then
```
Comment (daviddrysdale) [inline integration/functions.sh]: [Use](https://google.github.io/styleguide/shell.xml?showone=Testing_Strings#Testing_Strings) `-z` ?

File: integration/functions.sh [hunk 2]
```diff
@@ -66,23 +66,32 @@ pick_unused_port() {
   done
 }
 
-# kill_pid tries to kill the given pid, first softly then more aggressively.
+# kill_pid tries to kill the given pid(s), first softly then more aggressively.
```
[the remark this answers is not in the document]
Comment (AlCutter) [reply integration/functions.sh]: Didn't want to do that as it'd break the `ct_integration_test.sh`, could do it in a follow-up switch over CL?

**Comment by daviddrysdale** [comment]
There are also [other](https://github.com/google/certificate-transparency-go/blob/master/trillian/integration/ct_functions.sh#L240-L243) [users](https://github.com/google/trillian-examples/blob/master/gossip/integration/gossip_functions.sh#L160-L163) who could do with an update once this is in...

File: integration/functions.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (daviddrysdale) [reply integration/functions.sh]: Ah, good point, keep the name.

File: integration/functions.sh [hunk 3]
```diff
@@ -207,18 +216,20 @@ log_prep_test() {
 #  - RPC_SERVER_PIDS : bash array of RPC server pids
 #  - ETCD_PID        : etcd pid
 log_stop_test() {
```
[the remark this answers is not in the document]
Comment (AlCutter) [reply integration/functions.sh]: Good call, done.

File: integration/functions.sh (hunk 1 above)
Comment (AlCutter) [reply integration/functions.sh]: done

**Comment by AlCutter** [comment]
> There are also [other](https://github.com/google/certificate-transparency-go/blob/master/trillian/integration/ct_functions.sh#L240-L243) [users](https://github.com/google/trillian-examples/blob/master/gossip/integration/gossip_functions.sh#L160-L163) who could do with an update once this is in...

Yup, I know - they'll get theirs :)

**Pull request merged**